specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/ManagedServerDnsAliases.json

```diff
@@ -0,0 +1,365 @@
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "201": {
```

Comment:
>201 [](http://example.com/codeflow?start=11&length=3)

Where does the api return 201? Is it when the resource is created synchronously?

If 201 is returned for an asynchronous PUT, the response body must contain a provisioningState property.

[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#creatingupdating-using-put](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#creatingupdating-using-put)